node/tests/SharedTests.ts

```diff
@@ -1774,10 +1793,13 @@ export function runBaseTests(config: {
             await runTest(async (client: BaseClient) => {
                 const key = uuidv4();
                 const field = uuidv4();
-                const fieldValueMap = {
-                    [field]: "10",
-                };
-                expect(await client.hset(key, fieldValueMap)).toEqual(1);
+                const fieldValueList = [
```

Comment:
Now you can revert some tests, it will also show that old API still works

Reply:
I have reverted a few. I guess I can revert more tests. Reverting 5 more will have good number of tests for both input types.